packages/oscal-react-library/src/components/OSCALAnchorLinkHeader.tsx

```diff
@@ -12,7 +12,16 @@ const AnchorLinkIcon = styled(LinkIcon)(({ theme }) => ({
   color: theme.palette.grey[500],
 }));
 
-export default function OSCALAnchorLinkHeader(props) {
+export interface AnchorLinkProps {
+  urlFragment?: string;
+}
+
+export interface OSCALAnchorLinkHeaderProps {
+  children: any;
+  value?: any;
```

Comment:
I would suggest adding some documentation to these variables. I can help you out if you need any.

Reply:
@Bronstrom I would appreciate that! They are `any` and don't have documentation since it is not super clear what they should be :laughing:

Reply:
Here is the general idea of the two:

`children` - (`any`, but would suggest trying `ReactNode`) This is the text or element, which is most commonly a styled `Typography`, surrounded by an `OSCALAnchorLinkHeader`.
`value` - (`string` or `null`) This is a given value/name to be used in the id and fragment. When one of these isn't provided the text within `children` is used for the id and fragment.

Reply:
`conformLinkIdText` makes it so that `children` can not be a `ReactNode` I thought, right?

Reply:
`conformLinkIdText` returns a `string`, but it's one parameter is `any`. I'm pretty sure `children` here is `any`, and its being passed to `conformLinkIdText`.